app/src/main/java/com/example/fitjournal/MainActivity.kt

```diff
@@ -347,7 +366,7 @@ class MainActivity : ComponentActivity() {
                                             IconButton(onClick = {
                                                 navController.navigateUp()
                                                 navController.navigateUp()
-                                                journalEntryViewModel.save()
+//                                                journalEntryViewModel.save()
                                             }) {
```

Comment:
Is this intentionally left commented out?

Reply:
yea, not working on any of the crud functions as of yet